test/smoke.spec.ts

```diff
@@ -10,4 +10,17 @@ describe('smoke tests', () => {
     expect(stdout).toMatch(versionRegex);
     expect(stderr).toBe('');
   });
+
+  if (process.env.IMAGE_TAG === 'driftctl') {
```

Comment:
I'm not sure that this is the right way to achieve that. I wanted to run that test only on that image.

I've looked at `image-specific-tests` but I wasn't sure that it was to place to hook in:
It seems that we are building twice the `python-3.8` image just to run specific test on it, but does this job has been designed to run other specific image tests ?